src/TLuaInterpreter.cpp

```diff
@@ -1251,52 +1251,47 @@ int TLuaInterpreter::getMapEvents(lua_State * L){
     return 0;
 }
 
-int TLuaInterpreter::centerview( lua_State * L )
+int TLuaInterpreter::centerview(lua_State* L)
 {
-    Host * pHost = TLuaInterpreter::luaInterpreterMap[L];
-    if( ! pHost ) {
-        lua_pushnil( L );
+    Host* pHost = TLuaInterpreter::luaInterpreterMap[L];
+    if (!pHost) {
+        lua_pushnil(L);
```

Comment:
I think we finally decided to treat this as an error - someone ought to go through and convert all the remaining:

    lua_pushnil(L);
    lua_pushstring(L, "functionName: NULL Host pointer - something is wrong!");
    return 2;

to:

    lua_pushstring(L, "functionName: NULL Host pointer - something is wrong!");
    return lua_error(L);

!